### PR TITLE
fix wrong figure name

### PR DESCRIPTION
**Pull request by jhpb7**
The figure "names" was wrong. I changed it to the correct one.

### CODE REVIEW

File: docs/basics/101-127-yoda.rst [hunk 1]
```diff
@@ -234,7 +234,7 @@ compute outputs, and the code and outputs are captured,
 version-controlled, and linked to the input data. Each input data in turn
 is a (potentially nested) subdataset, but this is not visible
 in the directory hierarchy.
-Figure 3 in comparison emphasizes a process view on a project and
+Figure  in comparison emphasizes a process view on a project and
```
Comment (mslw) [inline docs/basics/101-127-yoda.rst:237]: Here, I'd keep the (corrected) figure number, too
```suggestion
Figure 6.2, in comparison, emphasizes a process view on a project and
```

**Review by mslw — CHANGES_REQUESTED** [review]
Thanks @jhpb7, well spotted. I left a small change suggestion that you can approve, and then we can merge.

File: docs/basics/101-127-yoda.rst (hunk 1 above)
Comment (jhpb7) [reply docs/basics/101-127-yoda.rst]: Hello @mslw, thanks for adding. Of course I approve.

**Comment by mslw** [comment]
Thanks again!

Feel free to add yourself to the zendo.json file in this repo - either by opening a pull request, or by leaving your affiliation, name, and optionally ORCID in this issue with a comment so that we can put your details in.

@all-contributors please add @jhpb7 for bug

**Comment by mslw** [comment]
@all-contributors please add @jhpb7 for bug

**Pull request merged**